skrub/_joiner.py

```diff
@@ -183,15 +191,13 @@ def transform(self, X: pd.DataFrame, y=None) -> pd.DataFrame:
             The final joined table.
         """
 
-        for pairs in self.tables:
-            # TODO: Add an option to fuzzy_join on multiple columns at once
-            # (will be if len(inter_col)!=0)
-            aux_table = pairs[0]
+        for df, cols in self.tables:
+            aux_table = df
             X = fuzzy_join(
                 X,
                 aux_table,
                 left_on=self.main_key,
-                right_on=pairs[1],
```

Comment:
I can't comment on it, but
```python
left_on=self.main_key,
```

should instead be:
```python
left_on=self.main_key_,
```
correct?

Reply:
You are right, there was a problem there. No need for this `self.main_key_` class instance